js/clinicalTimeline.js

```diff
@@ -874,7 +874,7 @@ window.clinicalTimeline = (function(){
 
   function getTickValues(beginning, ending, zoomLevel) {
       tickValues = [];
-      timePeriod = daysToTimeObject(difference(parseInt(beginning), parseInt(ending)));
+      daysPerMonth = 30;
       maxTime = daysToTimeObject(parseInt(ending));
```

Comment:
It would be better not to define your own daysPerMonth, but use the one from one of the time objects, e.g. `maxTime.daysPerMonth`. Also, as mentioned before I incorrectly made the scope here global, so better to change maxTime to `var maxTime` and `minTime` to `var minTime`. Thanks